src/core/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1,827 +1,842 @@
+    <value>Selected height</value>
+  </data>
+  <data name="ImageResizer_Formatting_SelectedWidth.Text" xml:space="preserve">
+    <value>Selected width</value>
+  </data>
+  <data name="ImageResizer_Formatting_Sizename.Text" xml:space="preserve">
+    <value>Size name</value>
+  </data>
+  <data name="FancyZones_MoveWindowsBasedOnPositionCheckBoxControl.Content" xml:space="preserve">
+    <value>Move windows based on their position</value>
+    <comment>Windows refers to application windows</comment>
+  </data>
+  <data name="GeneralSettings_NewVersionIsAvailable" xml:space="preserve">
+    <value>New update available</value>
+  </data>
+  <data name="GeneralSettings_VersionIsLatest" xml:space="preserve">
+    <value>PowerToys is up-to-date.</value>
+  </data>
+  <data name="FileExplorerPreview_IconThumbnail_GroupSettings.Text" xml:space="preserve">
+    <value>Icon Preview</value>
+  </data>
+  <data name="FileExplorerPreview_PreviewPane_GroupSettings.Text" xml:space="preserve">
+    <value>Preview Pane</value>
+  </data>
+  <data name="FileExplorerPreview_RunAsAdminRequired.Text" xml:space="preserve">
+    <value>You need to run as administrator to modify these settings</value>
+  </data>
+  <data name="FileExplorerPreview_AffectsAllUsers.Text" xml:space="preserve">
+    <value>The settings on this page affect all users on the system</value>
+  </data>
+  <data name="FileExplorerPreview_RebootRequired.Text" xml:space="preserve">
+    <value>A reboot may be required for changes to these settings to take effect</value>
+  </data>
+  <data name="FancyZones_ExcludeApps_TextBoxControl.PlaceholderText" xml:space="preserve">
+    <value>Example: outlook.exe</value>
+  </data>
+  <data name="ImageResizer_FilenameFormatPlaceholder.PlaceholderText" xml:space="preserve">
+    <value>Example: %1 (%2)</value>
+  </data>
+  <data name="ColorModeHeader.Text" xml:space="preserve">
+    <value>Choose a mode</value>
+  </data>
+  <data name="Radio_Theme_Dark.Content" xml:space="preserve">
+    <value>Dark</value>
+    <comment>Dark refers to color, not weight</comment>
+  </data>
+  <data name="Radio_Theme_Light.Content" xml:space="preserve">
+    <value>Light</value>
+    <comment>Light refers to color, not weight</comment>
+  </data>
+  <data name="Radio_Theme_Default.Content" xml:space="preserve">
+    <value>Windows default</value>
+    <comment>Windows refers to the Operating system</comment>
+  </data>
+  <data name="Windows_Color_Settings.Text" xml:space="preserve">
+    <value>Windows color settings</value>
+    <comment>Windows refers to the Operating system</comment>
+  </data>
+  <data name="ColorPicker_CopiedColorRepresentation.Header" xml:space="preserve">
+    <value>Copied color representation</value>
+  </data>
+  <data name="ColorPicker_OpenEditor.Content" xml:space="preserve">
+    <value>Opens editor, you can browse your colors history or pick a new color</value>

```

Comment:
**Suggestion 1:**
Open editor - You can browse your history of colors or pick a new color